core-bundle/src/Resources/contao/templates/elements/ce_vimeo.html5

```diff
@@ -6,7 +6,25 @@
     <?php if ($this->aspect): ?>
       <div class="responsive ratio-<?= $this->aspect ?>">
     <?php endif; ?>
-    <iframe<?= $this->size ?> src="<?= $this->src ?>" allowfullscreen></iframe>
+      <?php if ($this->videoSplash): ?>
+        <a id="videoSplash<?= $this->id ?>" class="image_container" href="<?= $this->src ?>">
```

Comment:
The `class="image_container"` does not make any sense any more, since that is only used on `figure` within Contao. I think I should remove it. If someone needs a class on the `<a>`, they can add it themselves via their own template.